tsconfig.json

```diff
@@ -17,7 +17,7 @@
     ],
     "paths": {
       "@/*": [
-        "src/*"
+        "src/*",
```

Comment:
Fun fact: the JSON parser `tsc` uses has the quirk of allowing trailing commas, which the JSON parser for -- say `package.json` doesn't have. In fact, `tsc` allows comments in its JSON too.